test/views/ViewSpec.scala

```diff
@@ -33,14 +33,16 @@ class ViewSpec extends WordSpec with MustMatchers with ViewTemplates with ViewMa
 
   implicit protected def htmlBodyOf(html: Html): Document = Jsoup.parse(html.toString())
 
-  private val messagesApi: MessagesApi = instanceOf[MessagesApi]
+  private val messagesApi: MessagesApi = ViewSpec.realMessagesApi
 
   protected implicit def messages(implicit request: Request[_]): Messages =
     new AllMessageKeysAreMandatoryMessages(messagesApi.preferred(request))
 
-  protected implicit def messages(key: String)(implicit request: Request[_]): String = messages(request)(key)
+  protected def messages(key: String, args: Any*)(implicit request: Request[_]): String = messages(request)(key, args: _*)
 
-  protected def journeyRequest(answers: Answers) = JourneyRequest(answers, AuthenticatedRequest(Operator(providerId), FakeRequest().withCSRFToken))
+  protected implicit val fakeRequest: Request[AnyContent] = FakeRequest().withCSRFToken
```

Comment:
Sure this isn't going to break a tonne of tests? Not sure we'd want this really

Reply:
It is used in some places, but eventually there are only 2. Reverted the change.